src/test/java/uk/gov/digital/ho/hocs/casework/api/CaseDataServiceTest.java

```diff
@@ -372,6 +370,26 @@ public void shouldGetCaseOnlyFilteredAdditionalData() throws ApplicationExceptio
         assertThat(result.getAdditionalFields().stream().noneMatch(f -> f.getLabel().equals("UnfilteredField")));
     }
 
+    @Test
+    public void shouldGetRegionUUIDForCase() {
+
+        Correspondent correspondent = new Correspondent(UUID.randomUUID(), "TYPE", "name",
+                new Address("postcode","address1","address2","address3","county"),
+                "phone", "email", "", "extKey");
+        //CaseData caseData = new CaseData(caseType, caseID, new HashMap<>(), objectMapper, caseReceived);
```

Comment:
We can remove this block (commented out)

Reply:
Commented code has been removed.